### ISSUE TITLE
MockGenerator doesn't install  on old-style package based projects

### ISSUE THREAD

**Issue by eredmo** [mocklis/mocklis#14]
When trying to add the MockGenerator nuget package I get the following:
```
Attempting to gather dependency information for package 'Mocklis.MockGenerator.1.0.0' with respect to project 'TestProject', targeting '.NETFramework,Version=v4.5'
Gathering dependency information took 49.21 ms
Attempting to resolve dependencies for package 'Mocklis.MockGenerator.1.0.0' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'Mocklis.MockGenerator.1.0.0'
Resolved actions to install package 'Mocklis.MockGenerator.1.0.0'
Adding package 'Mocklis.MockGenerator.1.0.0', which only has tools group, to project 'TestProject'
Adding package 'Mocklis.MockGenerator.1.0.0' to folder 'D:\Studies\Test\packages'
Added package 'Mocklis.MockGenerator.1.0.0' to folder 'D:\Studies\Test\packages'
Added package 'Mocklis.MockGenerator.1.0.0' to 'packages.config'
Executing script file 'D:\Studies\Test\packages\Mocklis.MockGenerator.1.0.0\tools\tools\install.ps1'...
Join-Path : Cannot find path 'D:\Studies\Test\packages\Mocklis.MockGenerator.1.0.0\tools\analyzers' because it 
does not exist.
At D:\Studies\Test\packages\Mocklis.MockGenerator.1.0.0\tools\tools\install.ps1:12 char:19
+ ... zersPaths = Join-Path (Join-Path (Split-Path -Path $toolsPath -Parent ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Studies\Test...tools\analyzers:String) [Join-Path], ItemNotFoundExce 
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.JoinPathCommand
 
Successfully installed 'Mocklis.MockGenerator 1.0.0' to TestProject
Executing nuget actions took 616.9 ms
Time Elapsed: 00:00:00.9195259
========== Finished ==========
```

**Comment by eredmo** [comment]
Issue is due to bug in installer package. Install scripts are in path tools\tools\ rather than tools\.

**Comment by eredmo** [comment]
Fix released as v1.0.1 and as part of new release v1.1.0.
